lib/repl.js

```diff
@@ -421,7 +421,10 @@ function REPLServer(prompt,
       }
     }
 
-    if (cmd || self.bufferedCommand) {
+    // self.context.setBreakpoint exists we are in the debugger.
```

Comment:
'If'?  I'm a bit uneasy about this check, it's not very robust.  Doing `setBreakPoint = 1` in the normal REPL will make this code think it's being called from the debugger.